packages/api/src/resolvers/highlight/index.ts

```diff
@@ -51,6 +52,7 @@ export const createHighlightResolver = authorized<
         highlightType: input.type || HighlightType.Highlight,
         highlightPositionAnchorIndex: input.highlightPositionAnchorIndex || 0,
         highlightPositionPercent: input.highlightPositionPercent || 0,
+        representation: input.representation || RepresentationType.Content,
```

Comment:
Yeah i think we will have to make this nullable and just default to the reader content representation otherwise all clients will need to be updated before we deploy this.

Reply:
The `input.representation` is nullable and defaults to `Content`

Reply:
thanks!